src/app-layout/visual-refresh/notifications.tsx

```diff
@@ -1,26 +1,41 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import clsx from 'clsx';
 import { highContrastHeaderClassName } from '../../internal/utils/content-header-utils';
 import { useAppLayoutInternals } from './context';
 import styles from './styles.css.js';
 import testutilStyles from '../test-classes/styles.css.js';
+import { useResizeObserver } from '@cloudscape-design/component-toolkit/internal';
 
 export default function Notifications() {
+  const { notifications } = useAppLayoutInternals();
+  if (!notifications) {
+    return null;
+  }
```

Comment:
@gethinwebster see this line. 

If there are no notifications all code down the line is not called

Reply:
Let's call it a refactoring for the cleanless sake then